src/components/fileviewer.js

```diff
@@ -40,10 +37,8 @@ export default class FileViewerContainer extends Component {
     /* get revision and path parameters from URL */
     const parsedQuery = queryString.parse(props.location.search);
     if (!parsedQuery.revision || !parsedQuery.path) {
-      this.setState({
-        status: {
+      this.status.setState({
           app: "Undefined URL query ('revision', 'path' fields are required)",
-        },
```

Comment:
The rest of the lines changed now change `this.status.state` instead of `this.state.status`